model/Licensing/Classes/ListedLicense.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# ListedLicense
+
+## Summary
+
+A license that is listed on the SPDX License List.
+
+## Description
+
+A ListedLicense represents a License that is listed on the SPDX License List
+at https://spdx.org/licenses.
+
+## Metadata
+
+- name: ListedLicense
+- SubclassOf: License
+- Instantiability: Concrete
+
+## Properties
+
+- listVersionAdded
+  - type: xsd:string
+  - minCount: 0
```

Comment:
why not mandatory?

Reply:
Because not every license on the license list has a value for this field, it was added later than the creation of the license list.